actions/get_contents.yaml

```diff
@@ -13,6 +14,10 @@ parameters:
     type: "string"
     description: "Repository name."
     required: true
+  base_url:
+    type: "string"
+    description: "GitHub API url, include /api/v3 for GitHub Enterprise."
+    required: false
```

Comment:
What happens if it's not specified?

Reply:
It uses the base_url set in the pack configuration.

Reply:
It would be less cases that multiple git systems are used at the same time.  So the most of cases, the base_url are not necessary to be set.